src/language/en/en.json

```diff
@@ -6,7 +6,11 @@
     "headerTitle": "Token Listings",
     "version": "Version",
     "footer": "Evodex is open source, get involved.",
-    "placeholder": "This Amount"
+    "placeholder": "This Amount",
```

Comment:
@tetogomez I think we should replace "This Amount" for  "Enter Amount"

Reply:
in develop branch it is the current value